### ISSUE TITLE
Predicted Probabilities are in negative real numbers range

### ISSUE THREAD

**Issue by surbhimadan92** [JeongsooP/Community-Forensics#4]
I have run your evaluation code on my set of images, what i observed it was giving the negative values and in real number range, which should not be the case for probability prediction. Can you please update is it returning logits value and we need to add sigmoid to get the probability?

**Comment by JiaLiangShen** [comment]
<img width="1160" height="926" alt="Image" src="https://github.com/user-attachments/assets/d6e4995b-c51e-475a-bb7d-9e56185be6d5" />

In the file model.py, the Sigmoid function has been used to output the probability distribution. Please check your codes, this repo works perfect for me.

**Comment by JeongsooP** [comment]
Sorry for the late reply! I was busy with life and had no time to check GitHub for the past few weeks.

Yes, the model outputs logits. If you made any changes to the code, you would have to manually add sigmoid to the output of the model to get the probabilities. (Thanks Jialiang for answering in my stead!)
